.github/workflows/publish.yml

```diff
@@ -85,3 +60,16 @@ jobs:
               run: npm publish
               env:
                   NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+
+            - name: Get merged pull request
+              id: getMergedPullRequest
+              run: |
+                read -r number < <(gh pr list --search ${{ github.sha }} --state merged --json 'number' | jq -r '.[0] | [.number] | join(" ")')
```

Comment:
this is copied from https://github.com/Expensify/App/blob/ce598bf02f08539edfc914aebe3f08afe518f891/.github/workflows/welcome.yml#L18-L19.

You can test it locally by:

- installing the `gh` cli if you don't have it already. `brew install gh`
- cd into the `react-native-onyx` repo
- run `read -r number < <(gh pr list --search ${{ github.sha }} --state merged --json 'number' | jq -r '.[0] | [.number] | join(" ")')` followed by `echo "|$number|"` (the bars show that there's no leading or trailing spaces)

Reply:
of course replace `github.sha` with a commit sha from this PR